  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0  0     0    0     0    0     0      0      0 --:--:--  0:00:01 --:--:--     0100  1043  100  1043    0     0    780      0  0:00:01  0:00:01 --:--:--   780
commit bc4ecf90b4585d535f2f3511fc0e910494309b35
Author: Harald Sitter <sitter@kde.org>
Date:   Fri Aug 1 16:35:16 2014 +0200

    don't construct bogus KAuthAction objects
    
    KAuthAction(0) actually calls the QString constructor which will dispatch
    bogus polkit auth checks, polkitqt and polkitd are not terribly happy about
    those and subsequently don't want to talk to the KCM anymore, even if it
    manages to create a proper Action
    
    creating a null Action rather than accidentally triggering the QString
    ctor ensures that we only do polkit interaction with reasonable actionids
    making sure that authentication works as expected
    
    REVIEW: 119540

diff --git a/src/kcmodule.cpp b/src/kcmodule.cpp
index 92e5427..9a3ae30 100644
--- a/src/kcmodule.cpp
+++ b/src/kcmodule.cpp
@@ -43,7 +43,7 @@ public:
         _useRootOnlyMessage(false),
         _firstshow(true),
         _needsAuthorization(false),
-        _authAction(0),
+        _authAction(),
         _unmanagedWidgetChangeState(false)
     { }
 
